core/bin/dust_api.rs

```diff
@@ -669,6 +758,7 @@ async fn main() -> Result<()> {
         .route("/", get(index))
         // Projects
         .route("/projects", post(projects_create))
+        .route("/projects/:project_id/clone", post(projects_clone))
```

Comment:
See above comment on naming. I think that `fork` is probably a better name here. Thoughts?